pkgdb2/ui/packages.py

```diff
@@ -1232,6 +1232,15 @@ def package_request_new():
         pkg_poc = flask.g.fas_user.username
         pkg_upstream_url = form.upstream_url.data
 
+        bz = APP.config.get('PKGDB2_BUGZILLA_URL')
+        if bz not in pkg_review_url:
+            try:
+                int(pkg_review_url)
```

Comment:
It'll blow but won't be able to convert the url to an integer and thus will use the input as provided